src/components/dashboard/table/index.tsx

```diff
@@ -0,0 +1,17 @@
+import TableMiu from '@mui/material/Table';
```

Comment:
for cleaner code, you can import Mui components in one  line no need to import every component in a single line